Adjust reference to inetconv(1M) to be inetconv(8) in the cups-lpd.8 man page.

This patch should be sent upstream.

--- man/cups-lpd.man.in.orig	2016-04-20 07:49:06.119469392 -0700
+++ man/cups-lpd.man.in	2016-04-20 07:49:28.518688055 -0700
@@ -113,7 +113,7 @@
 stray from this definition, remote status reporting to LPD
 clients may be unreliable.
 .SH SEE ALSO
-\fIcups(1)\fR, \fIcupsd(8)\fR, \fIinetconv(1m)\fR,
+\fIcups(1)\fR, \fIcupsd(8)\fR, \fIinetconv(8)\fR,
 \fIinetd(8)\fR, \fIxinetd(8)\fR,
 .br
 http://localhost:631/help
